.travis.yml

```diff
@@ -1,11 +1,21 @@
 language: r
 sudo: required
 cache: packages
+dist: trusty
 
 apt_packages:
-  - r-cran-rgdal
   - libproj-dev
 
+before_install:
+  - sudo add-apt-repository ppa:ubuntugis/ubuntugis-unstable --yes
+  - sudo apt-get --yes --force-yes update -qq
+  - sudo apt-get install --yes libudunits2-dev libproj-dev libgeos-dev libgdal-dev
+
+r_packages:
```

Comment:
Sorry, can you elaborate? I'm pretty inexperienced when it comes to Travis. These packages are all in the DESCRIPTION under Suggests, but don't I need them listed here to get them to install for testing?